firebase.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Since this project is only meant to use the [Cloud Firestore](https://firebase.google.com/docs/firestore), can you please remove the other local configuration options for the services which aren't being used? I want to confirm that we're testing things against an local environment that will match what we end up deploying.

Reply:
We have added this for testing purpose and will be added in .gitignore file.

Reply:
Since this file is in gitignor, please remove it from this PR